src/schema/field_type.rs

```diff
@@ -176,6 +193,7 @@ impl FieldType {
             FieldType::Facet(ref _facet_options) => true,
             FieldType::Bytes(ref bytes_options) => bytes_options.is_indexed(),
             FieldType::JsonObject(ref json_object_options) => json_object_options.is_indexed(),
+            FieldType::IpAddr(_) => false,
```

Comment:
Finish the renaming`Ip -> IpAddr` everywhere...

IpOptions --> IpAddrOptions
add_ip_field -> add_ip_addr_field

etc.